pkgs/build-support/fetchmavenartifact/default.nix

```
@@ -37,13 +37,13 @@ assert (url == "") || (urls == []);
 assert (repos != []) || (url != "") || (urls != []);
 
 let
-  name_ =
+  pname =
     lib.concatStrings [
-      (lib.replaceStrings ["."] ["_"] groupId) "_"
-      (lib.replaceStrings ["."] ["_"] artifactId) "-"
-      version
+      (lib.replaceStrings [ "." ] [ "_" ] groupId)
+      "_"
+      (lib.replaceStrings [ "." ] [ "_" ] artifactId)
     ];
```

```suggestion
  pname = (lib.replaceStrings [ "." ] [ "_" ] groupId) + "_" + (lib.replaceStrings [ "." ] [ "_" ] artifactId);
```
Is this equal to this?

---

looks to me equal yes. i will apply it.